src/components/dialogs/equipment-id/equipment-id-selector.js

```diff
@@ -111,14 +106,14 @@ export const EquipmentIdSelector = ({
             </Grid>
             <FormFiller lineHeight={fillerHeight}>
                 {fillerMessageId && !selectedValue && (
-                    <Box className={classes.message}>
+                    <Box sx={styles.message}>
                         <FormattedMessage id={fillerMessageId} />
                     </Box>
                 )}
                 <CircularProgress
                     // We keep the circular progress rendered but hidden to prevent an incomplete
                     // rendering when we set the choosen ID in the parent component.
-                    className={clsx({ [classes.hidden]: !selectedValue })}
+                    sx={mergeSx(!selectedValue && styles.hidden)}
```

Comment:
you don't need `mergeSx` here I think

Reply:
It is necessary to prevent a warning in the IDE : Type false | {color: string, width: number} is not assignable to type SxProps<Theme>
But I can change that to a ternary operator to solve this issue.

Reply:
ok, interesting to note that we can't give false to sx props thanks